server/models/Card.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+
+const cardSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: false,
+    },
+  },
+  { timestamps: true }
+);
+
+module.exports = { Card: mongoose.model("card", cardSchema), cardSchema };
```

Comment:
This will be adapted in the BE branch @street-jackal and I are working on

Reply:
Okay, when everything is merged in I will refactor for this to work with the new changes.